docs/modules/ROOT/pages/dataformats.adoc

```diff
@@ -50,7 +60,7 @@ local params = {
     "AutoEmptyElements": false
 };
 
-DS.Formats.write(payload, "application/xml", params);
+DS.Formats.write(payload.embeddedXMLValue, "application/xml", params);
```

Comment:
I think this is also a mismatch; the place where the example comes from has primed readers to believe it is a string containing XML, which couldn't then be written as XML